### PR TITLE
Improve README, LICENSE and package.json

### PR DESCRIPTION
**Pull request by koddr**
- Improve README for more readable format;
- Rename LICENSE by standard;
- Add `author` and `keywords` to package.json;

P.S. Thx for share your CSS reset — it's awesome! Love it 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,16 +9,40 @@ A tiny little reset that you can use as the basis of your CSS projects. You can
...
-  - Unpkg: `<link rel="stylesheet" href="https://unpkg.com/modern-css-reset/dist/reset.min.css" />`
-  - jsDelivr: `<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/modern-css-reset/dist/reset.min.css" />`
+- NPM:
 
-## Commands
+```console
+$ npm install --save-dev modern-css-reset
+```
+
+- Yarn:
```
Comment (Andy-set-studio) [inline README.md]: Let’s just get rid of the lists if we’re going to have proper code formatting 👍

File: README.md [hunk 2]
```diff
@@ -9,16 +9,40 @@ A tiny little reset that you can use as the basis of your CSS projects. You can
...
+```
+
+### Manual installation
 
 You can minify and move the main reset to the `dist` by running:
 
-```bash
-npm run build
+```console
+$ git clone https://github.com/hankchizljaw/modern-css-reset.git
```
Comment (Andy-set-studio) [inline README.md]: Having `$` here can be super confusing for beginners, so let's remove them.

Maybe do: 

You can minify and move the main reset to the `dist` by running:

```console
git clone https://github.com/hankchizljaw/modern-css-reset.git
```

then: 

```console
cd modern-css-reset
```

then: 

```console
npm run build
```

File: README.md [hunk 3]
```diff
@@ -9,16 +9,40 @@ A tiny little reset that you can use as the basis of your CSS projects. You can
...
+$ cd modern-css-reset
+$ npm run build
 ```
+
+## License
+
+MIT
```
Comment (Andy-set-studio) [inline README.md:60]: 👍

**Review by Andy-set-studio — CHANGES_REQUESTED** [review]
Thank you!

**Comment by koddr** [comment]
Okay, I changed README for your _point of view_ 😉

**Review by Andy-set-studio — APPROVED** [review]
Awesome, thank you!

**Pull request merged**